usr/share/rear/restore/VEEAM/default/500_restore.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Agent registration and restore command
+LogPrint "Deleting Veeam SQLite Linux Agent Database"
+rm -rf /var/lib/veeam/*
+
+LogPrint "Starting veeamservice agent for linux"
+systemctl start veeamservice
+StopIfError "Failed to start veeamservice Agent for Linux"
+sleep 5
+
+# Debug stuff
+#systemctl status veeamservice
+#veeamconfig mode info <if mode: Job, then run mode reset>
+
+LogPrint "Accept Veeam EULA agreement"
+# create directories for EULA agreement
+mkdir -p /usr/share/doc/veeam/
+touch /usr/share/doc/veeam/EULA
+touch /usr/share/doc/veeam/3rdPartyNotices.txt
+
+echo yes > /tmp/answer.txt
+echo yes >> /tmp/answer.txt
+
+LogPrint "Query available Veeam VBR Server"
+cat /tmp/answer.txt | veeamconfig vbrServer list
```

Comment:
Wouldn't be
```
echo -e "yes\nyes" | veeamconfig vbrServer list
```
simpler and more straightforward
than the indirection via /tmp/answer.txt
(or is /tmp/answer.txt also used somewhere else)?